srcopsmetrics/storage.py

```diff
@@ -97,8 +93,13 @@ class KnowledgeStorage:
     def __init__(self, is_local: Optional[bool] = False):
         """Initialize to behave as either local or remote storage."""
         self.is_local = is_local
+        location = os.getenv(StoragePath.LOCATION_VAR.value)
```

Comment:
Could we add `StoragePath.DEFAULT.value` here as a default value or do we want it to fail in this case?

Reply:
yes, this is true, no need to fail if we still have a default path